distributions/openhab/src/main/resources/bin/update.bat

```diff
@@ -0,0 +1,24 @@
+@ECHO off
+SETLOCAL
+
+IF [%~1]==[] GOTO printArgs
```

Comment:
I don't like this here. The whole point of the PS script was to handle autoupdates to the most current...

Reply:
Oversight - removing